tests/integration/test_integratord/data/test_cases/cases_integratord_read_json_alerts.yaml

```diff
@@ -0,0 +1,23 @@
+- name: 'Read valid json alert'
```

Comment:
Check the name convention from the other tests

Reply:
Ithis format is the same as in the VDT test suite refactor